public/app/plugins/datasource/opentsdb/components/FilterSection.tsx

```diff
@@ -121,20 +123,30 @@ export function FilterSection({
             return (
               <InlineFormLabel key={idx} width="auto" data-testid={testIds.list + idx}>
                 {fil.tagk} = {fil.type}({fil.filter}), groupBy = {'' + fil.groupBy}
-                <a onClick={() => editFilter(fil, idx)}>
+                <button type="button" className={clearButtonStyles(theme)} onClick={() => editFilter(fil, idx)}>
```

Comment:
I think we could've done smth like `const buttonStyles = useStyles2(clearButtonStyles)` at the top of and use that throughout the component instead of calling `clearButtonStyles` in multiple places.